Source/ReadmeBuilder.cs

```diff
@@ -11,13 +11,19 @@ internal class ReadmeBuilder
         StringBuilder md = new StringBuilder();
         List<List<string>> readmePrereqs = new List<List<string>>();
         List<List<string>> readme = new List<List<string>>();
-        string lastName = null;
+        List<PropertyName> columnNames = new List <PropertyName>();
 
         public ReadmeBuilder()
         {
 
         }
 
+        /// <summary>
+        /// Updates the main readme to display which model groups are being generated.
+        /// </summary>
+        /// <param name="executingAssembly"></param>
```

Comment:
are you planning on filling in the documentation for each parameter?

Reply:
I could if that would be helpful. @kcoley didn't in the Runtime code so I didn't think it was necessary.

Reply:
If you don't plan on filling it in, just remove them.